build-tests/prompt-agent.mdx

```diff
@@ -0,0 +1,134 @@
+
+<Frame caption="AI Agent informs about the progress of test step generation, 07/2024">
+  <img
+    src="/images/build/AI-Agent-generating-steps.png"
+    alt="AI Agent informs about the progress of test step generation"
+  />
+</Frame>
+
+If the AI agent signals a failed step with a yellow alert, click on `repair test` to help the Agent finish the generation successfully.
+
+<Frame caption="AI agent needs help your input to finalize generation - click repair test, 05/2025">
+  <img
+    src="/images/build/repair-yellow-test.png"
+    alt="AI agent needs your input to finalize generation - click repair test"
+  />
+</Frame>
+
+Follow the instuction of the Agent to repair the test.
+
+<Frame caption="Follow the instructions of the AI agent to repair and finalize test generation, 05/2025">
+  <img
+    src="/images/build/repair-mode.png"
+    alt="Follow the instructions of the AI agent to repair and finalize test generation"
+  />
+</Frame>
+
+## Pre-prompted login test
+
+If your app has a login functionality that wasn't AI auto-generated during set-up, we have a pre-prompted login test case for you. You can trigger it after giving us test user credentials.
+
+Go to `environments` menu item in the left project menu bar and click on the `default environment` in the list. You can have separate login test for separate environment.
+
+Select the `authentication` tab and add test user credentials we can use to generate the login test.
+
+<Tip>
+This test is also created on the fly for a **new project**, if we detect a login flow and you provide us with test user credentials.
+
+</Tip>
+
+<div class="mt-8" />
+
+<Frame caption="Provide test user credentials for login test generation in environment settings, 05/2025">
+  <img
+    src="/images/build/environments-default-settings.png"
+    alt="Provide test user credentials for login test generation in environment settings"
+  />
+</Frame>
+
+<div class="mt-8" />
+
+<Frame caption="Add test user credentials for login test generation in authentication tab of the environments settings, 05/2025">
+  <img
+    src="/images/build/pre-prompted-login-test.png"
+    alt="Add test user credentials for login test generation in authentication tab of the environments settings"
+    width="500"
+  />
+</Frame>
+
+Or check out this video instead:
+
+<iframe
+  width="560"
+  height="315"
+  src="https://www.youtube.com/embed/zUMeZ7sXdqc?si=a1nm8Pv_Hez3HNjK"
+  title="YouTube video player"
+  frameborder="0"
+  allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share"
+  referrerpolicy="strict-origin-when-cross-origin"
+  allowfullscreen
+></iframe>
+
+## Chain tests for faster test runs & maintenance
+
+A user flow is virtually a chain of test cases. When using the AI agent, you can use a dependency to chain test cases together. Shorter flows are faster, more specific and more reliable.
```

Comment:
Maybe rephrase the first sentence, imho "virtually a chain of test cases" doesn't make so much sense?